### PR TITLE
Demisto API branch

### PR DESCRIPTION
**Pull request by shalper**
# What changed

----

## Pages under review

- https://deploy-preview-455--logz-docs.netlify.app/api/#tag/Security-events

## Remaining work

- [ ] Technical review
- [ ] Copy Review

### CODE REVIEW

File: _source/api/logzio-public-api.yml [hunk 1]
```diff
@@ -2195,6 +2314,216 @@ definitions:
...
+        description: Tags are labels used to organize security rules.
+        example: threat
+        items:
+          type: object
+          example: { network_threat, recon}
+      hits:
+        type: integer
+        format: int32
+        description: Hits represent the number of logs that triggered the security rule before they were aggregated using the by `groupBy` condition.
+        example: 30
```
Comment (shalper) [inline _source/api/logzio-public-api.yml:2459]:
```suggestion
        description: Hits represent the number of logs that triggered the security rule before being aggregated by the `groupBy` condition.
        example: 30
```

File: _source/api/logzio-public-api.yml [hunk 2]
```diff
@@ -2195,6 +2314,216 @@ definitions:
...
+        description: Name of the security rule in Logz.io Cloud SIEM
+        example: AWS EC2 - Brute force SSH login attempts
+      description:
+        type: string
+        description: Typically an explanation of the security rule's logic and suggested next steps
+        example: Suggested next steps...
+      alertSummary:
+        type: string
+        description: Equivalent to the `condition` field in the rule
+        example: Alert if query '*' results GREATER_THAN_OR_EQUALS 5.00 in 10 minutes. Count on Group By '[userIdentity.userName, sourceIPAddress]'
```
Comment (yasmin-tr) [inline _source/api/logzio-public-api.yml:2410]: i'm really not sure how critical this is, and if anyone will notice, but the "group by" described here does not match the groupBy returned in below example . perhaps should be the same just for consistency? :)

File: _source/api/logzio-public-api.yml [hunk 3]
```diff
@@ -1908,7 +1922,112 @@ paths:
...
+        - Security events
+      operationId: searchSecurityRulesEvents
+      produces:
+        - application/json
+      parameters:
+        - in: body
+          name: body
+          required: false
+          schema:
+            $ref: '#/definitions/AlertsEventsSearchRequest'
```
[the remark this answers is not in the document]
Comment (shalper) [reply _source/api/logzio-public-api.yml:1944]: Would you prefer that I rename the definition?

**Comment by shalper** [comment]
Replaced by a more updated branch. Published.

**Pull request closed without merging**